RecoBTag/SecondaryVertex/python/candidateNegativeCombinedSecondaryVertexComputer_cfi.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoBTag.SecondaryVertex.candidateCombinedSecondaryVertexComputer_cfi import *
+
+candidateNegativeCombinedSecondaryVertexComputer = candidateCombinedSecondaryVertexComputer.clone()
+candidateNegativeCombinedSecondaryVertexComputer.vertexFlip = True
```

Comment:
Here I agree that cloning makes more sense. I'll see what I can do. Note, however, there are also cases where a PSet is one of the input parameters and later only one of the PSet's parameters would get modified. In those cases overwriting seems like a more elegant solution.